kinto/core/storage/postgresql/__init__.py

```diff
@@ -243,6 +243,9 @@ def create(self, collection_id, parent_id, record, id_generator=None,
         VALUES (:object_id, :parent_id,
                 :collection_id, (:data)::JSONB,
                 from_epoch(:last_modified))
+        ON CONFLICT (id, parent_id, collection_id) DO UPDATE
+        SET data = (:data)::JSONB,
+            last_modified = find_timestamp(:parent_id, :collection_id, from_epoch(:last_modified))
```

Comment:
I just read that `EXCLUDED.last_modified` would give us access to the rejected inserted value